tldw_Server_API/app/services/stripe_metering_service.py

```
@@ -6,19 +6,374 @@
+        try:
+            sub = await asyncio.to_thread(
+                stripe.Subscription.retrieve,
+                subscription_id,
+                expand=["items.data.price"],
+            )
+            for item in sub.get("items", {}).get("data", []):
+                price = item.get("price", {})
+                if price.get("recurring", {}).get("usage_type") == "metered":
+                    return item["id"]
+            # No metered item found — return None so caller can skip
+            return None
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)